subsys/tfm/boards/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+#-------------------------------------------------------------------------------
+# Copyright (c) 2020, Nordic Semiconductor ASA.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+#-------------------------------------------------------------------------------
+
+# NOTE: All code in this folder and subfolders is run as part of the TF-M
+#       build, not in a Zephyr context.
+
+cmake_policy(SET CMP0076 NEW)
```

Comment:
Should we comment why this is here?

Reply:
Looks like it is added to avoid a Cmake warning. All TFM platforms set this. I can add a comment if you want it.

Reply:
Removed